src/microsoft.sheets/Module.md

```diff
@@ -0,0 +1,100 @@
+This module allows users to connect to a Microsoft Office 365 Workbook located on Microsoft OneDrive.
```

Comment:
```suggestion
This module allows users to connect to a Microsoft Office 365 Workbook located on Microsoft OneDrive.
```
Shall we add hyperlinks to both "Microsoft Office 365 Workbook" and "Microsoft OneDrive".

Reply:
Done